src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by Francis on 21/9/2017.
+ */
+public interface Printable {
```

Comment:
Perhaps the header comment can be more informative than just stating the author?
Something along the lines of `Represents a printable entity.......`